src/library/scala/util/control/NonFatal.scala

```diff
@@ -12,7 +12,7 @@ package util.control
 /**
  * Extractor of non-fatal Throwables. Will not match fatal errors like `VirtualMachineError`
  * (for example, `OutOfMemoryError` and `StackOverflowError`, subclasses of `VirtualMachineError`), `ThreadDeath`,
- * `LinkageError`, `InterruptedException`, `ControlThrowable`.
+ * `LinkageError`, `InterruptedException`, `ControlThrowable`, AssertionError.
```

Comment:
tiny nit: missing backquotes around the added name

Reply:
oops, you're right. will fix shortly

Reply:
fixed now